whitepaper/chapters/blocks.tex

```diff
@@ -67,12 +67,12 @@ \section{Blocks}
 \subsection{Block Fields}
 
 \field{Height} is the block sequence number.
-The first block, called the \nind{nemesis block}, has a height of one.
+The first block, called the \nind{nemesis block}\index{block!nemesis}, has a height of one.
 Each successive block increments the height of the previous block by one.
 
 \field{Timestamp} is the number of milliseconds that have elapsed since the nemesis block.
 Each successive block must have a timestamp greater than that of the previous blocks because block time is strictly increasing.
-Each network tries to keep the average time between blocks close to \textit{target block time}.
+Each network tries to keep the average time between blocks close to \nind{target block time}.
```

Comment:
i wonder if we should replace all \emph with \nind ?

Reply:
this is probably bad idea, and, if desired, can be separate

Reply:
> i wonder if we should replace all \emph with \nind ?

not sure, some of them are under some existing 'group', i.e. \index{block!fee multiplier} and some of them are indexed *earlier*, I'm not sure if it has sense to index *all* occurrences of given term (especially that most of them will be on same or nearby page).